en/docs/Learn/DesignAPI/LifecycleManagement/extending-the-api-life-cycle.md

```diff
@@ -156,7 +156,7 @@ Following are some extension points that can be used:
 -   Change the execution code for each state transition
 
         !!! info
-    For all state transitions, the same execution class is used ( `                       org.wso2.carbon.apimgt.impl.executors.APIExecutor                     ` ). **** **** However, you can plug your own execution code when modifying the life cycle configuration. For example, if you want to add notifications for a specific state transition, you can plug your own custom execution class for that particular state in the API life cycle. Any changes are updated in the **Lifecycle** tab accordingly.
+    For all state transitions, the same execution class is used ( `                       org.wso2.carbon.apimgt.impl.executors.APIExecutor                     ` ). **** However, you can plug your own execution code when modifying the life cycle configuration. For example, if you want to add notifications for a specific state transition, you can plug your own custom execution class for that particular state in the API life cycle. Any changes are updated in the **Lifecycle** tab accordingly.
```

Comment:
Is **** correct?

Reply:
removed

Reply:
Fixed in Commit on Sep 9, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/d2c3ff8450dd3cdcef80a86027b482e4ec4f5410